pam/internal/adapter/newpasswordmodel.go

```diff
@@ -91,19 +93,28 @@ func (m newPasswordModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			return m, nil
 
 		case "tab":
+			m.errorMsg = ""
+			if m.focusIndex == 0 && m.passwordEntries[0].Value() != "" {
+				err := checkChallengeQuality(m.currentChallenge, m.passwordEntries[0].Value())
+				if err != nil {
+					m.Clear()
+					return m, sendEvent(errMsgToDisplay{msg: err.Error()})
+				}
+			}
+
+			cmds := []tea.Cmd{sendEvent(errMsgToDisplay{msg: ""})}
```

Comment:
why Is this needed?

Reply:
To reset the error message, otherwise it will keep showing even when the user already typed a valid password.